### ISSUE TITLE
Add project detail page using ajax calls to load each project step

### ISSUE THREAD

**Issue by calebsmith** [python/raspberryio#36]
- [ ] An ajax view for obtaining the data for a given project step
- [ ] JavaScript code for calling the ajax view
- [ ] JavaScript template to display project step html

**Comment by calebsmith** [comment]
On second thought, we might implement this with a django template instead. The back end would then render the html and send that in the ajax response. This may seem needless or wasteful but it would allow for an easy template based cache solution.

Alternatively, I think we could introduce a cacheing layer for the ajax response itself and still use a JavaScript template. In other words, the ajax view would attempt to find the JSON data in a cache, and otherwise make the necessary queries

**Comment by gsaines** [comment]
What is left to do on this one?

**Comment by calebsmith** [comment]
Absolutely everything unfortunately. I made this ticket because I'd love to add the feature but I fear it could take 25 hours or so. I think I'll flag it for phase 2 unless there are any objections

**Comment by gsaines** [comment]
Yes, I think this is most appropriately categorized as a phase 2 feature, thanks for labeling it already.
